### ISSUE TITLE
SEO - Pagination Issue?

### ISSUE THREAD

**Issue by shuffgy** [Sebobo/Shel.Blog#12]
Hey,
i'm using some tool for seo optimization and i get an error, that i can not explain to myself.

There is a page called "Blog" and one called "Hilfe" (Help). On both i use the blog plugin.

On each page i have currently 2 Posts.
But i get Crawling-Error for the blog-pagination for following URLS

https://schul.cloud/hilfe?--blog-pagination[currentPage]=2
https://schul.cloud/blog?--blog-pagination[currentPage]=2
https://schul.cloud/hilfe?--blog-pagination[currentPage]=3
https://schul.cloud/blog?--blog-pagination[currentPage]=3
https://schul.cloud/hilfe?--blog-pagination[currentPage]=4
https://schul.cloud/blog?--blog-pagination[currentPage]=4
[ and so on until ]
https://schul.cloud/blog?--blog-pagination[currentPage]=69
https://schul.cloud/hilfe?--blog-pagination[currentPage]=71

I dont know how and why so many URLs are generated and how to correct that. Because on every page i get 

> An exception was thrown while Neos tried to render your page
page<Neos.Neos:Page>/ body<Neos.Fusion:Template>/ content/ main<Neos.Neos:PrimaryContent>/ BlogFeed<Neos.Fusion:Matcher>/ element<Shel.Blog:BlogFeedHtml><Shel.Blog:BlogFeedHtml>/

I'm using Version 3.0 of shel/blog.

**Comment by Sebobo** [comment]
That's because of a evil `FIXME` in `Page.fusion` ;-) 
I didn't implement the check for when the meta `nextLink` should not be rendered anymore.

I think I now actually have an idea how to fix it with not so much effort.

**Comment by Sebobo** [comment]
@shuffgy should be solved with 3.1.1

**Comment by shuffgy** [comment]
wanted to try this out, but get some errors, sadly.
Did something else change between 3.0.8 and 3.1.1?

I need to check that, i only have some templatePath changes in my root.fusion for 
```
prototype(Shel.Blog:BlogEntry) < prototype(Content) {
	templatePath = 'resource://hkm.schulcloud/Private/Templates/Plugins/BlogEntry.html'
}
prototype(Shel.Blog:BlogFeed) < prototype(Content) {
	templatePath = 'resource://hkm.schulcloud/Private/Templates/Plugins/BlogFeed.html'
}
prototype(Shel.Blog:BlogFeedEntry) < prototype(Shel.Blog:BlogEntry) {
    templatePath = ${'resource://hkm.schulcloud/Private/Templates/Plugins/BlogFeedEntry.html'}
}
```

I dont even use the <footer> part in my BlogEntry.html
```
        <footer>
            <fusion:render path="socialButtons"/>
            <fusion:render path="disqus"/>

            <f:render partial="EntryPager"
                      arguments="{previousEntry: previousEntry, nextEntry: nextEntry, currentEntry: node, pagerArguments: pagerArguments}"/>
        </footer>

```

![73431396-3ec7-4401-abed-24d1dd178419](https://user-images.githubusercontent.com/17125448/39799873-24924ef2-5366-11e8-9cea-e38fb1d7aec1.png)

**Comment by Sebobo** [comment]
Sorry there was a missing dependency, thanks for finding it!
Should be solved with 3.1.2

**Comment by Sebobo** [comment]
Btw you should also check your date formatting, I improved the localizing of the date a lot but it could happen that something might look different.
